hw/dv/verilator/cpp/sv_scoped.h

```diff
@@ -48,4 +49,7 @@ class SVScoped {
   svScope prev_scope_;
 };
 
+// helper function to join two, possibly relative, scopes correctly.
+std::string join_sv_scopes(const std::string &a, const std::string &b);
```

Comment:
Maybe make this a static method of SVScoped? No need to add more stuff to the global namespace if we can avoid it.